Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -12343,6 +12342,133 @@ end
 				},
 			},
 		},
+		cUUID =
+		{
+			Desc = [[
+				Class representing a Universally Unique Identifier.
+				Functions taking a cUUID will also accept a string convertible to a UUID by FromString.
```

Comment:
This might need some emphasizing. How about:
```Note that all Cuberite's API functions that take a cUUID parameter will also accept a string in its place, as long as that string can be converted to a cUUID (using the {{#FromString_1|cUUID:FromString}} function).```
(I didn't check the FromString link for validity, please make sure it works :)